CHANGELOG.unreleased.md

```diff
@@ -33,3 +33,5 @@ Section headings should be at level 3 (e.g. `### Added`).
 - Correctly use `WANDB_CONFIG_DIR` for determining system settings file path (@jacobromero in https://github.com/wandb/wandb/pull/9711)
 - Prevent invalid `Artifact` and `ArtifactCollection` names (which would make them unloggable), explicitly raising a `ValueError` when attempting to assign an invalid name. (@tonyyli-wandb in https://github.com/wandb/wandb/pull/8773)
 - Prevent pydantic `ConfigError` in Pydantic v1 environments from not calling `.model_rebuild()/.update_forward_refs()` on generated types with ForwardRef fields (@tonyyli-wandb in https://github.com/wandb/wandb/pull/9795)
+- Files deleted for runs queried via `api.Runs(...)` no longer raises `wandb.errors.errors.CommError` (@jacobromero in https://github.com/wandb/wandb/pull/9748)
```

Comment:
It wasn't clear to me what raises an error here until I read through the full PR. You can also just use `CommError` instead of the fully qualified name in this case as the extra path doesn't add much:

> Calling `file.delete()` on files for runs queried via `api.Runs()` no longer raises `CommError`